content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md

```diff
@@ -168,212 +165,161 @@ To define the `Config` trait for the proof-of-existence pallet:
-   ```
-
-1. Add the `sp-std` crate to the list of features.
-
-   ```toml
-   [features]
-   default = ['std']
-   std = [
-       # -- snip --
-       'sp-std/std',
-   ]
-   ```
-
-1. Save your changes and close the file.
-
 ## Include pallet errors
 
 The events you defined indicate when calls to the pallet have completed successfully.
 Errors indicate when a call has failed, and why it has failed.
 For this tutorial, you define the following error conditions:
 
-- An attempt to claim a proof that has already been claimed.
+- An attempt to make a claim that has already exists.
 
-- An attempt to revoke a proof that does not exist.
+- An attempt to revoke a claim that does not exist.
 
-- An attempt to revoke a proof that has been claimed by another account.
+- An attempt to revoke a claim that is owned by another account.
 
 To implement the errors for the proof-of-existence pallet:
 
 1. Open the `pallets/template/src/lib.rs` file in a text editor.
 
 1. Replace the `#[pallet::error]` line with the following code block:
 
-   ```rust
-   #[pallet::error]
-   pub enum Error<T> {
-   	/// The proof has already been claimed.
-   	ProofAlreadyClaimed,
-   	/// The proof does not exist, so it cannot be revoked.
-   	NoSuchProof,
-   	/// The proof is claimed by another account, so caller can't revoke it.
-   	NotProofOwner,
-   }
-   ```
+  ```rust
+  #[pallet::error]
+  pub enum Error<T> {
+    /// The claim already exists.
+    AlreadyClaimed,
+    /// The claim does not exist, so it cannot be revoked.
+    NoSuchClaim,
+    /// The claim is owned by another account, so caller can't revoke it.
+    NotClaimOwner,
+  }
+  ```
 
 1. Save your changes.
 
 ## Implement a storage map for stored items
 
-To add a new proof to the blockchain, the proof-of-existence pallet requires a storage mechanism.
-To address this requirement, you can create a [hash map](https://en.wikipedia.org/wiki/Hash_table) that maps each proof to its owner and records the block number when the proof was made.
-To create this hash map, you can use the FRAME [`StorageMap`](https://paritytech.github.io/substrate/master/frame_support/storage/trait.StorageMap.html) trait.
+To add a new claim to the blockchain, the proof-of-existence pallet requires a storage mechanism.
+To address this requirement, you can create a key-value map, where each claim points to the owner and the block number when the claim was made.
+To create this key-value map, you can use the FRAME [`StorageMap`](https://paritytech.github.io/substrate/master/frame_support/pallet_prelude/struct.StorageMap.html).
 
 To implement storage for the proof-of-existence pallet:
 
 1. Open the `pallets/template/src/lib.rs` file in a text editor.
 
 1. Replace the `#[pallet::storage]` line with the following code block:
 
-   ```rust
-   #[pallet::storage]
-   pub(super) type Proofs<T: Config> = StorageMap<_, Blake2_128Concat, Vec<u8>, (T::AccountId, T::BlockNumber), ValueQuery>;
```

Comment:
Removed `ValueQuery` because `AccountId` does not impl default anymore. Switched to use hash too instead of vec